docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
+        │   └───.terraform ## NEW LOCAL DIRECTORY GENERATED FROM INITIALIZING REMOTE STATE
+        └───backend-state-setup
+    ```
+
+### **Step 5:** Deploy Cobalt's [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md)
+
+This is the step that demonstrates what the [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md) has to offer by deploying and creating actual Azure cloud resources. This infrastructure template creates an [ARM Resource Group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview), an [App Service Plan](https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans) and [App Service](https://docs.microsoft.com/en-us/azure/app-service/) running a public docker container. The previous step of initializing a Terraform environment from within the Azure Hello World CIT directory makes deployment possible.
+
+1. From the az-hello-world directory, execute the following commands to orchestrate a deployment.
+
+    ```bash
+    # See what terraform will try to deploy without actually deploying
+    terraform plan
+
+    # Execute a deployment
+    terraform apply
+    ```
+
+1. Take note of the name of the resource group output from Terraform apply.
+
+* **Azure Hello World CIT [Main.tf](https://github.com/microsoft/cobalt/blob/master/infra/templates/az-hello-world/main.tf) File**
+
+    A template is made up of custom modules and has several files. The main file that drives this template's deployment lives in the Main.tf file:
+
+    ```HCL
+    resource "azurerm_resource_group" "main" {
+        name     = local.app_rg_name
+        location = local.region
+    }
+
+    module "service_plan" {
+        source              = "../../modules/providers/azure/service-plan"
+        resource_group_name = azurerm_resource_group.main.name
+        service_plan_name   = local.sp_name
+    }
+
+    module "app_service" {
+        source                           = "../../modules/providers/azure/app-service"
+        app_service_name_prefix          = local.app_svc_name_prefix
+        service_plan_name                = module.service_plan.service_plan_name
+        service_plan_resource_group_name = azurerm_resource_group.main.name
+        docker_registry_server_url       = local.reg_url
+        app_service_config               = local.app_services
+    }
+    ```
+
+### **Step 6:** Validate Infrastructure Deployed Successfully
+
+If you correctly completed the previous step, you can trust that Azure resources are now living in Azure. However, let's take it a step further and get your eyes on the infrastructure that was just deployed. Seeing the end result gives you the full quick start experience and is something that should be done for all future Cobalt CIT deployments.
+
+1. Login to the Azure Portal.
+1. Search for "Resource Group" to find the Resource Group menu.
+1. Find and Select the name of the Resource Group output from the deployment.
```

Comment:
define "output"

Reply:
Removed "output". It could cause confusion with Terraform template outputs and also the term is too generic like "environment".